test/unit/tools/punchlist.rb

```diff
@@ -4,11 +4,12 @@ module Tools
       # Test for the 'bigfiles' tool within the quality gem
       module Punchlist
         def punchlist_expected_args
-          "--glob '{Rakefile,Dockerfile,{*,.*}.{md,rb,rake,gemspec,swift,cpp," \
-          'c,java,py,clj,cljs,scala,js,yml,sh,json},{src,app,config,db,lib,' \
-          'test,spec,feature}/**/{*,.*}.' \
-          '{md,rb,rake,gemspec,swift,cpp,c,java,py,clj,cljs,' \
-          "scala,js,yml,sh,json}}' " \
+          "--glob '{Dockerfile,Rakefile,{*,.*}.{c,clj,cljs,cpp,gemspec," \
+          'java,js,json,md,py,rake,rb,scala,sh,swift,' \
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.